applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp

```diff
@@ -420,31 +438,32 @@ Parameters ComputeHessianSolMetricProcess::GetDefaultParameters() const
 {
     Parameters default_parameters = Parameters(R"(
     {
-        "minimal_size"                        : 0.1,
-        "maximal_size"                        : 10.0,
+        "minimal_size"                         : 0.1,
+        "maximal_size"                         : 10.0,
         "sizing_parameters":
         {
-            "reference_variable_name"          : "DISTANCE",
-            "boundary_layer_max_distance"      : 1.0,
-            "interpolation"                    : "constant"
+            "reference_variable_name"              : "DISTANCE",
+            "boundary_layer_max_distance"          : 1.0,
+            "interpolation"                        : "constant"
         },
-        "enforce_current"                     : false,
+        "enforce_current"                      : false,
         "hessian_strategy_parameters":
         {
-            "metric_variable"                  : ["DISTANCE"],
+            "metric_variable"                      : ["DISTANCE"],
             "estimate_interpolation_error"         : false,
             "interpolation_error"                  : 1.0e-6,
             "mesh_dependent_constant"              : 0.28125
         },
-        "anisotropy_remeshing"                : true,
+        "anisotropy_remeshing"                 : true,
+        "enforce_anisotropy_relative_variable" : false,
```

Comment:
To me setting this to `false` by default is completely correct. However, since previously the "default" behavior was the corresponding to `true`, would you add a deprecation warning or similar?

Reply:
Tests were isotropic, I don't know, @AlejandroCornejo were you using this?

Reply:
I haven't used the anisot so OK